<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoogleCloudEngine</class>
 <widget class="QWidget" name="GoogleCloudEngine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>408</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpAuth">
     <property name="title">
      <string>Authentication</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="labelServiceJSON">
        <property name="text">
         <string>Service Account's Private Key JSON:</string>
        </property>
        <property name="buddy">
         <cstring>serviceJSON</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="KUrlRequester" name="serviceJSON"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnLogin">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labHelp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>102</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;p&gt;You need to &lt;a href=&quot;https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount&quot;&gt;create a Service Account&lt;/a&gt; in Google Cloud Console and provide public/private key JSON file below.&lt;/p&gt;&lt;p&gt;The Service Account needs to be assigned &lt;span style=&quot; text-decoration: underline;&quot;&gt;Cloud Translation API User&lt;/span&gt; role on IAM page or must be granted access to the Google's Cloud Translation APIs in a different manner.&lt;/p&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="lineHelp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labSource">
        <property name="text">
         <string>Source language:</string>
        </property>
        <property name="buddy">
         <cstring>langSource</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labTranslation">
        <property name="text">
         <string>Translate to:</string>
        </property>
        <property name="buddy">
         <cstring>langTranslation</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="langSource"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="langTranslation"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
